pkgs/applications/editors/kakoune/plugins/openscad.kak.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation {
+  name = "openscad.kak";
+  version = "unstable-2020-11-16";
+
+  src = fetchFromGitHub {
+    owner = "mayjs";
+    repo = "openscad.kak";
+    rev = "d9143d5e7834e3356b49720664d5647cab9db7cc";
+    sha256 = "0j4dqhrn56z77hdalfdxagwz8h6nwr8s9i4w0bs2644k72lsm2ix";
+  };
+
+  installPhase = ''
+    mkdir -p $out/bin $out/share/kak/autoload/plugins/
```

Comment:
Is it really needed to create empty `$out/bin` folder?

Reply:
Copy paste error, thanks :(